openshift/auth.app.yaml

```diff
@@ -49,16 +49,16 @@ objects:
               secretKeyRef:
                 name: auth
                 key: db.password
-          - name: AUTH_KEYCLOAK_REALM
+          - name: AUTH_OAUTH_SERVICE_REALM
             valueFrom:
               secretKeyRef:
                 name: auth
-                key: keycloak.realm
-          - name: AUTH_KEYCLOAK_SECRET
+                key: oauth.service.realm
```

Comment:
if we change the secret keys here, then we'll need to change the secrets that are deployed on prod and prod-preview.